src/core/caniuse.js

```diff
@@ -0,0 +1,193 @@
+      content = hyperHTML`<a href="${
+        "https://caniuse.com/#feat=" + caniuse.feature
+      }">caniuse.com</a>`;
+    }
+    resolve(content);
+  });
+  const definitionPair = hyperHTML.bind(document.createDocumentFragment())`
+    <dt class="caniuse-title"
+      id="${`caniuse-${caniuse.feature}`}">Can I Use this API?</dt>
+    <dd class="caniuse-stats">${{
+      any: contentPromise,
+      placeholder: "Fetching data from caniuse.com...",
+    }}</dd>`;
+  headDlElem.appendChild(definitionPair);
+  await contentPromise;
+}
+
+/**
+ * Normalizes `conf.caniuse` object to hold normalized configuration
+ * @param {Object} conf   configuration settings
+ */
+function normalizeConf(conf) {
+  const DEFAULTS = {
+    maxAge: 24 * 60 * 60 * 1000, // 24 hours (in ms)
+    browsers: ["chrome", "firefox", "safari", "edge"],
+    versions: 4,
+  };
+  if (typeof conf.caniuse === "string") {
+    conf.caniuse = { feature: conf.caniuse, ...DEFAULTS };
+    return;
+  }
+  if (Array.isArray(conf.caniuse.browsers)) {
+    conf.caniuse.browsers = conf.caniuse.browsers
+      .map(b => b.toLowerCase())
+      .filter(isValidBrowser);
+  } else {
+    conf.caniuse.browsers = DEFAULTS.browsers;
+  }
+  Object.assign(conf.caniuse, DEFAULTS, { ...conf.caniuse });
+  function isValidBrowser(browser) {
+    if (BROWSERS.has(browser)) {
+      return true;
+    }
+    pub(
+      "warn",
+      `Ignoring invalid browser "\`${browser}\`" in [\`respecConfig.caniuse.browsers\`](https://github.com/w3c/respec/wiki/caniuse)`
+    );
+    return false;
+  }
+}
+
+/**
+ * get stats for canIUse table
+ * @param {Object} caniuseConf    normalized respecConfig.caniuse
+ * @return {Object} Can I Use stats
+ * @throws {NetworkError} on failure
+ */
+async function fetchAndCacheJson(caniuseConf) {
+  const { apiURL, feature, maxAge } = caniuseConf;
+  const url = apiURL
+    ? apiURL.replace("{FEATURE}", feature)
+    : `${GH_USER_CONTENT_URL}${feature}.json`;
+  const request = new Request(url);
+  const response = await fetchAndCache(request, { maxAge });
+  const { stats } = await response.json();
+  return stats;
+}
+
+/**
+ * get HTML element for the canIUse support table
+ * @param  {Object} stats     CanIUse API results
+ * @param  {Object} conf      respecConfig.caniuse
+ */
+function createTableHTML(conf, stats) {
+  // render the support table
+  return hyperHTML`
+    ${conf.browsers.map(browser =>
+      addBrowser(browser, conf.versions, stats[browser])
+    )}
+    <a href="${`https://caniuse.com/#feat=${conf.feature}`}"
+      title="Get details at caniuse.com">More info
+    </a>`;
+
+  /**
+   * add a browser and it's support to table
+   * @param {String} browser      name of browser (as in CanIUse API response)
+   * @param {Number} numVersions  number of old browser versions to show
+   * @param {Object} browserData  stats data from api response
+   */
+  function addBrowser(browser, numVersions, browserData) {
+    if (!browserData) return "";
```

Comment:
Was testing, and this causes a bug because hyperHTML is expecting DOM Nodes. So, these need to be filtered out. Just `return;`  here instead.

Reply:
should add this as a test also?